bodge/TF_L1D1_tb_writer.vhd.bodge

```diff
@@ -0,0 +1,15 @@
+  -- A bodge for TrackBuilder to write TF concatenated track+stub data for L1L2.
+  -- (Needed to compare with emData/).
+  writeTF_L1D1_300 : entity work.FileWriterFIFO
+  generic map (
+    FILE_NAME  => FILE_OUT_TF&memory_enum_to_string(L1D1)&outputFileNameEnding,
+    FIFO_WIDTH  => 300
+  )
+  port map (
+    CLK => CLK,
+    DONE => FT_DONE,
+    WRITE_EN => TW_L1D1_stream_A_write,
```

Comment:
@aryd Since the TrackBuilder can output invalid tracks, you should use the valid bit of the track word to enable writing:
```suggestion
    WRITE_EN => (TW_L1D1_stream_A_write and TW_L1D1_stream_AV_din(103)),
```

Reply:
Fixed.